package.json

```diff
@@ -18,14 +18,18 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.3",
+    "@types/react-icons": "^3.0.0",
     "clsx": "^1.2.1",
+    "framer-motion": "^10.10.0",
     "html-webpack-plugin": "^5.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.8.0",
     "react-redux": "^8.0.5",
+    "react-responsive": "^9.0.2",
```

Comment:
Correct me if I am wrong, but are you aware that you never used this package anywhere in your code?

Same thing for the tailwind-scrollbar.

Please don't include un-used / un-necessary packages.

Reply:
ooo sure, I was looking how it workand I forget to remove it sorry